src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -30,29 +97,83 @@ public void setAimMode(int m) {
 
   public void cycleAimModeUp() {
     aimMode++;
-    if(aimMode > 3){aimMode = 0;}
+    if (aimMode > 3) {
+      aimMode = 0;
+    }
   }
 
   public void cycleAimModeDown() {
     aimMode--;
-    if(aimMode < 0){aimMode = 3;}
+    if (aimMode < 0) {
+      aimMode = 3;
+    }
   }
+
   public double getTY() {
-    //Gets TY, the vertical angle of the target from the limelight
-    return NetworkTableInstance.getDefault().getTable("limelight").getEntry("ty").getDouble(0);
+    // Gets TY, the vertical angle of the target from the limelight
+    return Math.toDegrees(NetworkTableInstance.getDefault().getTable("limelight").getEntry("ty").getDouble(0));
   }
+
   public double getDistance() {
-    //Uses Limelight to find distance to High Goal
-    return (Constants.kGoalHeight - Constants.kLLHeight) / Math.tan(getTY() + Constants.kLLAngle); //Return distance in feet
+    // Uses Limelight to find distance to High Goal
+    return (Constants.kHighHeight - Constants.kLLHeight) / Math.tan(getTY() + Constants.kLLAngle); // Return distance in
```

Comment:
math.tan requires radians.  https://docs.oracle.com/javase/8/docs/api/java/lang/Math.html#tan-double-

https://github.com/AusTINCANsProgrammingTeam/common-lib/blob/main/src/main/java/hardware/Limelight.java#L59

Reply:
Does that mean we need to redo our calculations, or should I just convert to degrees after .tan is run

Reply:
Link to the common code has an example:
            Math.tan(Math.toRadians(mLimelightMountingAngle + mLimelightTable.getEntry("ty").getDouble(0.0))); 

So...For your code:
    return (Constants.kHighHeight - Constants.kLLHeight) / Math.tan(Math.toRadians(getTY() + Constants.kLLAngle)); // Return distance in